alephmarcreader/abstractalephmarcreader.py

```diff
@@ -267,13 +295,20 @@ def get_author(self):
             if "aut" in person.roles:
                 author.append(person)
 
```

Comment:
Shouldn't there be a check for the filed 110 ("110 = Haupteintragung unter einem Körperschaftsnamen") as well?

Reply:
Technically, yes. But in BIBB, field 110 has never been used. In the instances, where the author is an institution, that institution is named in field 710 with the role 'aut' (see e.g. [000058439](https://aleph.unibas.ch/F/4BNNT8PNYCPGK8CICMQA9MPE6194BYLHD5VJEIGQN271FFG6IF-08311?func=full-set-set&set_number=672222&set_entry=000001&format=001), field `710 2`).  
So there's really no point in checking an unused field.

Reply:
Ok, got it!